### PR TITLE
BL-881: Highlight thumbnail for the current book

### PR DESCRIPTION
**Pull request by phillip-hopper**

### CODE REVIEW

File: src/BloomExe/CollectionTab/LibraryListView.cs [hunk 1]
```diff
@@ -21,7 +21,8 @@ namespace Bloom.CollectionTab
 {
 	public partial class LibraryListView : UserControl
 	{
-		private const int ButtonHeight = 110;
+		private const int ButtonHeight = 112;
+		private const int ButtonWidth = 92;
```
Comment (phillip-hopper) [inline src/BloomExe/CollectionTab/LibraryListView.cs]: Height and width increased so that the area available for text will be the same.

**Comment by hatton** [comment]
Phil, you highlight the button when you click on it. How about other ways to get a selected book, like making a new one? Consider whether OnBookSelectionChanged() could be used instead of the click to set the highlighted book.

**Comment by hatton** [comment]
Phil, you'll probably want to rebase now that #387 is merged.

**Comment by phillip-hopper** [comment]
Pull request updated.

**Pull request merged**